src/components/ActiveCard.jsx

```diff
@@ -52,9 +54,32 @@ class ActiveCard extends React.Component {
     this.setState({ editBox });
   }
 
+  onDelete() {
+    if (this.props.preferences.preferences && this.props.preferences.preferences.cribsheet) {
+      const cribCopy = this.props.preferences.preferences.cribsheet.slice();
```

Comment:
Is this safe or should this check for the existence of `this.props.preferences.preferences.cribsheet` before running `.slice()`?

Reply:
It appears to be checking the existence. Am I missing something?

Reply:
Wait, derp, ignore my comment, I am obviously missing the obvious `if()` at the top there. Sorry, speed reading was a bad idea.